src/main/java/org/asamk/signal/json/JsonAttachmentData.java

```diff
@@ -0,0 +1,3 @@
+package org.asamk.signal.json;
+
+public record JsonAttachmentData(String dataBase64) {}
```

Comment:
I don't think base64 needs to be part of the name.

```suggestion
public record JsonAttachmentData(String data) {}
```

Reply:
This was changed